### ISSUE TITLE
Likes no longer show in Interactions tab; only mentions

### ISSUE THREAD

**Issue by ldmosquera** [TwidereProject/Twidere-Android#642]
Roughly since the time that the Twitter website separated the Notifications tab into "All" and "Mentions", Twidere stopped showing Likes along with Mentions in the Interactions tab.

I've reinstalled the app and cleared caches and it does not help; API keys are as they come by default, and I did not check the "Mentions Only" checkbox in the Interactions tab.

**Comment by mariotaku** [comment]
It's impossible to display likes and RTs unless we use official keys.

**Comment by ldmosquera** [comment]
I followed the [suggestions in the wiki](https://github.com/TwidereProject/Twidere-Android/wiki/Changing-Consumer-Key-Secret) and used my API keys, but no dice - `Mentions only` is still checked and grayed out.

I also tried with the [Official Android keys here](https://gist.github.com/shobotch/5160017) and still no dice.

Did you mean there is absolutely no way, or were either of these things supposed to work?

Thanks for the help :)

**Comment by mariotaku** [comment]
Official keys should work, I'll check that out

**Comment by ldmosquera** [comment]
An update: I tried with [Tweetdeck keys from this page](https://gist.github.com/shobotch/5160017); after removing and readding the Interactions tab, `Mentions only` WAS selectable, and I set it to off, but I still don't get likes in that tab.

**Comment by mariotaku** [comment]
@ldmosquera is Twitter for Android key works?

**Comment by 1011X** [comment]
@ldmosquera If you're still having trouble with likes not showing up in the interactions tab, they finally showed up for me after I messed around with the settings. To be clear, I had mentions working for me, but likes were not showing up at all.

Before doing all this, I went through the process of setting-up the official keys on Twidere, then removing my account from the app and re-adding it. When that's done, make sure you have the interactions tab open. From the interactions tab, open the left pane, click on "Settings", go to the "Tabs" section, and click on "Interactions". A window with 2 checkbox options should appear. One is "Your following only" and the other is "Mentions only".

First, I tried clicking on the "Mentions only" option, saving, and having Twidere restart itself. This took me back to the interactions tab, but I didn't notice any changes. In fact, when I went back to the window with the 2 options, the "Mentions only" option *unchecked itself*. I found that kinda weird, but I kept going.

I tried the same thing and chose the "Your following only" option. My memory gets kinda fuzzy from here. I saved and Twidere restarted, just like last time, except this time it didn't show notifications from people I don't follow. I think after that I tried swiping downwards to refresh the interactions tab, which resulted in some of my recent likes finally showing up, but only the likes from those I follow. After going back to the setting again and unchecking the "Your following only" option, saving, and restarting, the tab then showed the rest of the likes that were missing.

Hope that helps.

**Comment by ldmosquera** [comment]
Thanks a lot for the detailed description @1011X -- I'm pretty sure I did that many times in the past, but I just tried once more and IT WORKED.

I uninstalled the app, then installed the latest version from F-Droid store (3.4.48), then set [official Twitter for Android keys](https://gist.github.com/shobotch/5160017), then deleted+readded account, and likes work showed up right away in the Interactions tab.

FINALLY I can go back to this excellent client!! No more official client BS -- I could dance of happyness.

Thanks again @1011X, and @mariotaku :)
